### PR TITLE
docs: update readme

### PR DESCRIPTION
**Pull request by FabienMotte**
This PR updates the README based on the changes made in #415.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -60,8 +58,11 @@ i.async=1,i.src=n,c.parentNode.insertBefore(i,c)
 
 #### 2. Initialize the library
 
+Initializing the library is optional, as you can specify the [credentials for each event](#sending-events-to-multiple-algolia-applications) when sending them. This gives you the flexibility to manage your Algolia credentials on a per-event basis, without having to configure them upfront.
+
 ```js
 aa('init', {
+  // `appId` and `apiKey` are optional
```
Comment (dhayab) [inline README.md:65]: Nitpick: we could follow the structure set in the next comment for consistency.

```suggestion
// Optional: configure default Algolia credentials for events
aa('init', {
```

File: README.md [hunk 2]
```diff
@@ -70,19 +71,20 @@ aa('init', {
...
-| Option            | Type           | Default                  | Description                                    |
-| ----------------- | -------------- | ------------------------ | ---------------------------------------------- |
-| **`appId`**       | `string`       | None (required)          | The identifier of your Algolia application     |
-| **`apiKey`**      | `string`       | None (required)          | The search API key of your Algolia application |
-| `userHasOptedOut` | `boolean`      | `false`                  | Whether to exclude users from analytics        |
-| `region`          | `'de' \| 'us'` | Automatic                | The DNS server to target                       |
+| Option            | Type           | Default                  | Description                                     |
+| ----------------- | -------------- | ------------------------ | ----------------------------------------------- |
+| **`appId`**       | `string`       | None                     | The identifier of your Algolia application.     |
+| **`apiKey`**      | `string`       | None                     | The search API key of your Algolia application. |
```
Comment (dhayab) [inline README.md:77]:
```suggestion
| `appId`           | `string`       | None                     | The identifier of your Algolia application.     |
| `apiKey`          | `string`       | None                     | The search API key of your Algolia application. |
```

**Review by dhayab — APPROVED** [review]
Good for me, feel free to apply or discard the suggestions I left.

File: README.md (hunk 2 above)
Comment (FabienMotte) [reply README.md:77]: Nice catch, thanks!

**Pull request merged**